Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/webapp/WebAppSettingModel.java

```diff
@@ -48,6 +50,8 @@
     private String region = "";
     private String pricing = "";
     private JavaVersion jdkVersion = JavaVersion.JAVA_8_NEWEST;
+    private RuntimeStack linuxRuntime = RuntimeStack.TOMCAT_8_5_JRE8;
+    private OperatingSystem os;
```

Comment:
Why not assign a default value for consistency?

Reply:
@Matthew-Dong Any suggestions for the default value? Linux or Windows?